### ISSUE TITLE
WCAG - IDs used in ARIA and labels must be unique - FormGroupText Description

### ISSUE THREAD

**Issue by fchaulandadasa** [digitalnsw/nsw-design-system-react#17]
I'm checking my code generated via this library against WCAG 2.0 standards using Axe plugin on Chrome
Axe raise a critical alert on the Description of an input using `FormGroupText`

```js
    <FormGroupText
        as="input"
        status={validateField(
            formik.touched.WorksApproval,
            formik.errors.WorksApproval,
            formik.values.WorksApproval
        )}
        htmlId="WorksApproval"
        name="WorksApproval"
        label="Works Approval"
        helper="Water supply work approval number (Format: 00AA000000)"
        value={formik.values.WorksApproval}
        statusText={formik.errors.WorksApproval ? formik.errors.WorksApproval : formik.touched.WorksApproval ? 'OK' : ''}
        onBlur={formik.handleBlur}                            
        onChange={formik.handleChange}
        disabled={loading}
    />
```                            

To solve this, this span is supposed to have a unique ID : 

https://github.com/digitalnsw/nsw-design-system-react/blob/87657c4e1a20a4ba373145a7c5984d1c93385721/src/component/forms/group-elements/index.js#L21

by probably re-using the same logic of the parent `<span>`

**Comment by fchaulandadasa** [comment]
After investigation the problem comes from the parrent `span`

In the `example` folder you can generate the example website.
It will render this part:
```html
<div class="nsw-form__group ">
<label for="id1" class="nsw-form__label ">Your name</label>
<span id="helperid1" class="nsw-form__helper   undefined">
<span id="icon-id1" class="material-icons nsw-material-icons" focusable="false" aria-hidden="true"></span>as it is written on your drivers' licence</span>
<input class="nsw-form__input " aria-invalid="false" aria-describedby="helperid1" type="text" id="id1">
<span id="helperid1" class="nsw-form__helper   nsw-form__helper--valid undefined">
<span id="icon-check_circleid1" class="material-icons nsw-material-icons" focusable="false" aria-hidden="true">check_circle</span>
Valid drivers licence
</span>
</div>
```

`helperid1` is duplicated at it re-use twice the same component. One for the description, one for the message feedback

![image](https://user-images.githubusercontent.com/85094380/175816372-bbbdc962-a7b2-4441-a98e-6c7ffb4cb347.png)


https://github.com/digitalnsw/nsw-design-system-react/blob/87657c4e1a20a4ba373145a7c5984d1c93385721/src/component/forms/group-elements/index.js#L20

This part should be changed : 
```
id={`${status === 'invalid' ? 'error' : 'helper'}${htmlId}`}
```
